src/bp/admin/ui/public/fonts/font-roboto.css

```diff
@@ -0,0 +1,128 @@
+/* cyrillic-ext */
+@font-face {
```

Comment:
Is this the same font-face that we use in the studio ? Seems like we sometimes use lato and sometimes roboto do you know where and why this differs ?

Reply:
We use roboto everywhere, not sure why "lato" is specified somewhere on the studio, I guess it will be removed eventually